_blueprints/work_item_types.adoc

```diff
@@ -0,0 +1,78 @@
+= Work Items and Work Item Types (Proposal)
+This is a working document to come up with a system for describing and defining types of work items. 
+
+== Vision
+There is a central work item (micro-)service that serves as a common platform for storage and query of work items. Client services extend the basic work item types to offer business functionality like different ways of planning (scrum, waterfall, etc.), testing, ideation, etc. These service in turn are the basis for our UI.
+
+== Goals
+Why do we even have such a thing like a work item type? Why not just store arbitrary json structures? For one, it allows us to optimize storage of work items, for example setting indices on fields in the work items. Work item types also restrict the work items to have a form which we can store efficiently. Work item types can also be queried, as in "give me all work items of type X". This will allow clients to only select the work items they can work with and lets them assume presence of needed fields. 

```

Comment:
There are a few more reasons for WorkItemType that comes to mind;

* Defines structure
* Defines display names (possible display form)
* Defines state transition model
* Allows validation

Essentially works as a Schema for what you can do with this WorkItem. Storage etc is an impl detail.